File: mRemoteNG/Connection/Connection.Protocol.Telnet.vb
Namespace Connection
    Namespace Protocol
        Public Class Telnet
            Inherits Connection.Protocol.PuttyBase

            Public Sub New()
                Me.PuttyProtocol = Putty_Protocol.telnet
            End Sub

            Public Enum Defaults
                Port = 23
            End Enum
        End Class
    End Namespace
End Namespace

File: mRemoteV1/Connection/Protocol/deprecated/Connection.Protocol.Telnet.cs


namespace mRemoteNG.Connection.Protocol
{
	public class Telnet : PuttyBase
	{
				
		public Telnet()
		{
			this.PuttyProtocol = Putty_Protocol.telnet;
		}
				
		public enum Defaults
		{
			Port = 23
		}
	}
}
